recipes-wpe/wpeframework/include/webkitbrowser.inc

```diff
@@ -1,48 +1,273 @@
-# Provides the WPE WebKitBrowser plugin settings for WPEFramwork
-# Also extends WPE instances in the form of UX and a Apps specific instance
-
-WEBKITBROWSER_AUTOSTART ?= "true"
-WEBKITBROWSER_MEDIADISKCACHE ?= "false"
-WEBKITBROWSER_MEMORYPRESSURE ?= "databaseprocess:50m,networkprocess:100m,webprocess:300m,rpcprocess:50m"
-WEBKITBROWSER_MEMORYPROFILE ?= "128m"
-WEBKITBROWSER_MSEBUFFERS ?= "audio:2m,video:15m,text:1m"
-WEBKITBROWSER_STARTURL ?= "http://localhost:8080/index.html"
-WEBKITBROWSER_USERAGENT ?= "Mozilla/5.0 (Macintosh, Intel Mac OS X 10_11_4) AppleWebKit/602.1.28+ (KHTML, like Gecko) Version/9.1 Safari/601.5.17"
-WEBKITBROWSER_DISKCACHE ?= "0"
-WEBKITBROWSER_XHRCACHE ?= "false"
-WEBKITBROWSER_TRANSPARENT ?= "false"
-WEBKITBROWSER_THREADEDPAINTING ?= "1"
-WEBKITBROWSER_WEBINSPECTOR ?= "0.0.0.0:9998"
-WEBKITBROWSER_USER ?= ""
-WEBKITBROWSER_GROUP ?= ""
-WEBKITBROWSER_RESOLUTION ?= "720p"
-WEBKITBROWSER_THREADEDPAINTING ??= ""
+# Provides the WPE WebKitBrowser plugin settings
+# Also extends WPE instances in the form of specific apps instance like Apps, HTML apps, UX etc
+
+WPE_WEBKITBROWSER_AUTOSTART ??= "true"
+WPE_WEBKITBROWSER_RESUMED ??= "true"
+WPE_WEBKITBROWSER_STARTURL ??= "http://localhost:8080/index.html"
+WPE_WEBKITBROWSER_USERAGENT ??= "Mozilla/5.0 (Macintosh, Intel Mac OS X 10_11_4) AppleWebKit/602.1.28+ (KHTML, like Gecko) Version/9.1 Safari/601.5.17"
+WPE_WEBKITBROWSER_LOCALSTORAGE_ENABLE ??= "true"
+WPE_WEBKITBROWSER_WEBINSPECTOR ??= "0.0.0.0:9998"
```

Comment:
Can we be consistent with the naming and name that one as WPE_WEBKITBROWSER_WEBINSPECTOR_ADDRESS ?

Reply:
Done. It is variable inside the recipe scope, so there is no additional change for backward compatibility